x-pack/plugins/index_lifecycle_management/server/routes/api/templates/register_fetch_route.ts

```diff
@@ -40,11 +36,6 @@ function filterAndFormatTemplates(templates: {
       continue;
     }
     const formattedTemplate = {
-      index_lifecycle_name:
```

Comment:
Good to remove it if not needed. I do wonder why it was there in the first place. Do you know?

Reply:
I'm not sure why it had all that properties there, but my guess is some of ES API has changed in the meantime and new things were added instead of replacing old stuff. And maybe there were plans to show more info in UI, as you suggested with attached policies etc.